.sandbox_ui.yaml

```diff
@@ -0,0 +1,6 @@
+host: localhost
+port: 8080sdaf
+welcome_info: |
+  This web app is part of an implementation study.
+  It only contains basic functionalities and shouldn't be used
+  for production.
```

Comment:
Does it make sense to even maintain this file?
It would only be used if you run the service/the ui directly (without the devcontainer setup). And then running multiple services simultaneously would result in clashing port numbers anyway, because every service uses port 8080.

Reply:
We can get rid of this if required.

Reply:
Yes, if it is confusing with the file with `.devcontainer` then go ahead and remove it.